pkgs/build-support/setup-hooks/compress-man-pages.sh

```diff
@@ -10,14 +10,13 @@ compressManPages() {
 
     # Compress all uncompressed manpages.  Don't follow symlinks, etc.
     find "$dir"/share/man/ -type f -a '!' -regex '.*\.\(bz2\|gz\|xz\)$' -print0 \
-        | while IFS= read -r -d $'\0' f
-    do
-        if gzip -c -n "$f" > "$f".gz; then
-            rm "$f"
+    | xargs -0 -P "$NIX_BUILD_CORES" -I {} sh -c '
+        if gzip -c -n "{}" > "{}.gz"; then
+            rm "{}"
```

Comment:
We can just do this and call it a day:

```suggestion
    | xargs -0 -P "$NIX_BUILD_CORES" gzip
```

Reply:
gzip will already remove the original file when it's done.

Reply:
If it throws an actual error we might actually want to know about it why. The most likely cause I would see is disk full and I don't thing we want to ignore that error.